### PR TITLE
dedupe contour resources using dekupe tool

### PR DESCRIPTION
**Pull request by dprotaso**
Fixes: #333

- Introduce a tool in our release script that dedupes k8s resources

/kind cleanup

**Release Note**

```release-note
NONE
```

**Docs**

```docs
NONE
```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -3,6 +3,7 @@ module knative.dev/net-contour
 go 1.14
 
 require (
+	github.com/dprotaso/dekupe v0.0.0-20201119194102-ec62861e5660
```
Comment (tcnghia) [inline go.mod:6]: Can this lives in our Knative automation repo  or test-infra?

cc @chaodaiG @n3wscott @mattmoor

**Comment by dprotaso** [comment]
Yeah just let me know

**Comment by dprotaso** [comment]
Actually - I worry taking a dependency on `test-infra` since we just dropped it to avoid a circle loop.

File: go.mod (hunk 1 above)
Comment (n3wscott) [reply go.mod:6]: Sure. What does the tool do?

Comment (dprotaso) [reply go.mod:6]: it dedupes k8s objects in yaml

**Comment by dprotaso** [comment]
I dropped the vendoring since `go_run_tool` should just work

**Review by tcnghia — APPROVED** [review]
/lgtm

**Pull request merged**